src/main/java/frc/robot/constants/CharacterizationConstants.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.constants;
+
+public class CharacterizationConstants {
+    public final double initialPower;
+    public final double powerIncrement;
+    public final int cycleCount;
+    public final double cycleLength;
+    public final double tolerancePercentage;
+
+    /**
+     * @param initialPower        Starting power between -1 and 1 used in the initial state.
```

Comment:
i could write it is so:
1. it could work either way. so systems that move can start with a negative value.
2. I could set it to Math.abs() and keep the docs
3. set it to Math.abs() but write between 0 and 1

Reply:
so what do you suggest?